Signal/src/call/CallService.swift

```diff
@@ -249,6 +249,18 @@ protocol CallServiceObserver: class {
         return  "CallServiceActiveCallNotification"
     }
 
+    class func presentCallInterstitialNotificationName() -> String {
+        return  "PresentCallInterstitialNotification"
+    }
```

Comment:
For strings accessed in multiple classes like this, I like to prefix the class name, i.e. "CallServicePresentCallInterstitialNotification"